lib/active_record_doctor/detectors/missing_non_null_constraint.rb

```diff
@@ -19,7 +19,11 @@ class MissingNonNullConstraint < Base # :nodoc:
       private
 
       def message(column:, table:)
-        "add `NOT NULL` to #{table}.#{column} - models validates its presence but it's not non-NULL in the database"
+        message = "add `NOT NULL` to #{table}.#{column}"
+        unless ["created_at", "updated_at"].include?(column)
+          message = "#{message} - models validates its presence but it's not non-NULL in the database"
+        end
+        message
```

Comment:
`if/else` will look easier for understanding, imo. 

Also rails considers `created_on`/`updated_on` columns the same as with `_at` suffix.

Reply:
Thanks for the review @fatkodima. I have applied your suggestion and also added support for created_on / updated_on.